qiskit/pulse/instructions/frequency.py

```diff
@@ -60,3 +60,32 @@ def channel(self) -> PulseChannel:
         scheduled on.
         """
         return self._channel
+
+
+class ShiftFrequency(Instruction):
+    r"""Shift the channel frequency away from the LO frequency."""
+    def __init__(self, frequency: float,
+                 channel: Optional[PulseChannel],
+                 name: Optional[str] = None):
+        """Creates a new shift frequency instruction.
+
+        Args:
+            frequency: Frequency shift of the channel in Hz.
+            channel: The channel this instruction operates on.
+            name: Name of this set channel frequency command.
+        """
+        self._frequency = float(frequency)
+        self._channel = channel
+        super().__init__((frequency, channel), 0, (channel,), name=name)
+
+    @property
+    def frequency(self) -> float:
+        """New frequency."""
```

Comment:
this docstring should be updated (the frequency here is the delta from the set frequency, not the actual output frequency for the channel)

Reply:
Fixed in [e8dd486](https://github.com/Qiskit/qiskit-terra/pull/4390/commits/e8dd48689b1a2502e19f59785619a63f4de82d25).